configs/errors.tslint.json

```diff
@@ -45,6 +45,12 @@
     "no-trailing-whitespace": true,
     "no-unused-expression": true,
     "no-var-keyword": true,
+    "no-void-expression": {
+      "severity": "warning",
```

Comment:
Don't we want to raise the `warning` to `error` now that the violators are fixed?
You can remove the `"severity": "warning"` here since the whole file treats these rules as `errors`.

Reply:
Yes I will do that. Thank you for reminding ;)